jormungandr-integration-tests/build.rs

```diff
@@ -5,17 +5,16 @@ use std::env;
 
 fn main() {
     // generate grpc mock
-    /*
-    protoc_rust::run(protoc_rust::Args {
+
+    /*protoc_rust::run(protoc_rust::Args {
```

Comment:
Is there a problem with running protoc automatically so that this needs to be commented out?

If you do, also add the below line to this function to avoid excessive rebuilds:

```rust
    println!("cargo:rerun-if-changed=../../chain-libs/network-grpc/proto/node.proto");
```

Reply:
I think the problem is with CI, which doesn't have protoc installed:
```
Caused by:
  process didn't exit successfully: `/mnt/crate/target/debug/build/jormungandr-integration-tests-c4c28dc0b4a16e82/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'protoc: Custom { kind: NotFound, error: "failed to spawn `\"protoc\" \"--version\"`: No such file or directory (os error 2)" }', src/libcore/result.rs:1165:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

https://circleci.com/gh/input-output-hk/jormungandr/14884

Reply:
Filed #1020 about this problem.